pkg/api/util_test.go

```diff
@@ -0,0 +1,40 @@
+package api_test
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+var (
+	UpdateGolden = flag.Bool("update-golden", false,
```

Comment:
There is an interesting talk from Mitchell Hashimoto about this: https://speakerdeck.com/mitchellh/advanced-testing-with-go?slide=19